README.md

```diff
@@ -2,145 +2,174 @@
 
-Node.js has great frameworks 💚, they never meant to be production ready - Practica.js bridges this gap. Based on your preferred framework, we generate a small code that demonstrates a full flow, from API to DB, that is packed with good practices. For example, we incude a hardened dockerfile, N-Tier folder structure, great testing templates, and more. This can save a great deal of time and painful mistakes. All our decisions are being made, [thoughtfully and neatly documented](/docs/decisions), we strive to keep things as simple and standard as possible and base our work on the popular guide ['Node.js best practices'](https://github.com/goldbergyoni/nodebestpractices)
+<br/>
+
+# What's in here? An Overview
+
+Although Node.js has great frameworks 💚, they were never meant to be production ready immediately. Practica.js aims to bridge the gap. 
+
+Based on your preferred framework, we generate some example code that demonstrates a full workflow, from API to DB, that is packed with good practices. For example, we include a hardened dockerfile, N-Tier folder structure, great testing templates, and more. This saves a great deal of time and can prevent painful mistakes. All decisions made are [neatly and thoughtfully documented](/docs/decisions). 
+
+We strive to keep things as simple and standard as possible and base our work off the popular guide: [Node.js Best Practices](https://github.com/goldbergyoni/nodebestpractices)
 
 <br/>
 
-# `Table of contents`
-
-- [`A very quick start`](#a-very-quick-start)
-- [`Our philosophy and unique value`](#our-philosophy-and-unique-value)
-- [`Example applications (place holder, not ready yet)`]()
-  - [Express, Postgresql, with common best practices](https://github.com/practicajs/practica/blob/main/docs/not-ready-yet.md)
-  - [Express, Mongodb, with common best practices](https://github.com/practicajs/practica/blob/main/docs/not-ready-yet.md)
-  - [Express, Postgresql, with all best practices (advanced)](https://github.com/practicajs/practica/blob/main/docs/not-ready-yet.md)
-  - [Minimal with project setup configuration only](https://github.com/practicajs/practica/blob/main/docs/not-ready-yet.md)
-  <details><summary>More flavours</summary>
-  
-  - [soon] fastify, postgresql
-  
-  - [soon] fastify, mongo
-  
-  - [Generate your own interactively]()
-  
-  - More coming soon
-  </details>
-
-- [`Practices and features`](#practices-and-features)
-- [`Documentation site (coming soon)`](https://github.com/practicajs/practica/blob/main/docs/not-ready-yet.md)
-- [`YouTube channel (soon)`](https://github.com/practicajs/practica/blob/main/docs/not-ready-yet.md)
-- [`The people behind this project`](#the-people-behind-this-project)
+# `Table of Contents`
+
+- [`Installation and Usage`](#installation-and-usage)
```

Comment:
Would prefer the "Very quick" phrasing (what we changed)

Reply:
Gotcha. 

How attached to 'very quick' are we? It's difficult to make a title with the words 'very quick'. Not much works well with them unless it's in a full sentence. If we're wanting to emphasize the speed at which someone can get started, maybe we can do something with alliteration and call it a 'Speedy Setup' or, more generalized, something like 'Quick Start' / 'Rapid Start'. 

What do you think?

Reply:
My belief is that the visitor should read _unusual_ words. It should not sound too bad and lexically wrong, but we should find wording that is unusual and bold

I trust your judgment, put something, don't be afraid to deviate a bit from 'standard' English (e.g., Super-quick setup)